parameters.toml

```diff
@@ -0,0 +1,110 @@
+# General protocol parameters.
+[parameters]
+native_token = "NAAN"
+# Minimum number of blocks in an epoch.
+# This should be 10 hours given that blocks are on average 10 seconds. ( (10*60*60)/10 )
+min_num_of_blocks = 3_600
+# Maximum expected time per block (in seconds).
+max_expected_time_per_block = 10
+# Max payload size, in bytes, for a tx.
+max_tx_bytes = 1048576
+# Max payload size, in bytes, for a tx batch proposal.
+max_proposal_bytes = 6291456
+# vp allowlist
+vp_allowlist = []
+# tx allowlist
+tx_allowlist = []
+# Implicit VP WASM name
+implicit_vp = "vp_implicit"
+# Expected number of epochs per year (also sets the min duration of an epoch in seconds)
+# Epochs should be 12 hours. ( (365*24*60*60)/(12*60*60) )
+epochs_per_year = 730
+# Maximum number of signature per transaction
+max_signatures_per_transaction = 15
+# Max gas for block
+max_block_gas = 20000000
+# Fee unshielding gas limit
+fee_unshielding_gas_limit = 20000
+# Fee unshielding descriptions limit
+fee_unshielding_descriptions_limit = 15
+
+# Map of the cost per gas unit for every token allowed for fee payment
+[parameters.minimum_gas_price]
+nam = "0.000001"
+
+# Proof of stake parameters.
+[pos_params]
+# Maximum number of active validators.
+max_validator_slots = 257
+# Pipeline length (in epochs). Any change in the validator set made in
+# epoch 'n' will become active in epoch 'n + pipeline_len'.
+pipeline_len = 2
+# Unbonding length (in epochs). Validators may have their stake slashed
+# for a fault in epoch 'n' up through epoch 'n + unbonding_len'.
+# 3 days for unbonding
+unbonding_len = 6
+# Votes per fundamental staking token (namnam)
+tm_votes_per_token = "1"
+# Reward for proposing a block.
+block_proposer_reward = "0.125"
+# Reward for voting on a block.
+block_vote_reward = "0.1"
+# Maximum inflation rate per annum (10%)
+max_inflation_rate = "0.0"
+# Targeted ratio of staked tokens to total tokens in the supply
+# start with 0 inflation
+target_staked_ratio = "0.0"
```

Comment:
Nit: I think the only important thing for issuing 0 inflation is the `max_inflation_rate` be equal to "0.0" as it currently is. Definitely the comment about starting with 0 inflation should be above `max_inflation_rate`

Reply:
And the nominal value for `target_staked_ratio` is 0.667